<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TypeLogementAdmin</class>
 <widget class="QDialog" name="TypeLogementAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/projetimages/projetimages/e.png</normaloff>:/projetimages/projetimages/e.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView {
selection-background-color:red;
border-radius:15px;
}

QDialog
{
		background-image: url(:/projetimages/projetimages/a.jpg);

}
*{
font-family:centry gothlic ;
font-size:15px;
}
QWidget#widget
{

	background:rgba(0,0,0,0.7);
border-radius:15px;
}
 


QPushButton
 {
     background-color: red;

     border-radius: 10px;

 }


QPushButton
{
color:white;

border-raduis:15px;
}
QPushButton:hover
{

color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}

</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>431</width>
     <height>401</height>
    </rect>
   </property>
   <widget class="QTableView" name="tableViewtypelogadmin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterlog_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
